manifests/charts/ztunnel/templates/daemonset.yaml

```diff
@@ -263,11 +263,15 @@ spec:
           value: "{{ $val }}"
         {{- end }}
         {{- end }}
+        - name: ISTIO_META_CLUSTER_ID
+          value: {{ .Values.multiCluster.clusterName | default "Kubernetes" }}
         volumeMounts:
         - mountPath: /var/run/secrets/istio
           name: istiod-ca-cert
         - mountPath: /var/run/secrets/tokens
           name: istio-token
+        - mountPath: /etc/istio/config
+          name: istio-ztunnel-config
```

Comment:
is this supposed to map to https://github.com/istio/ztunnel/blob/master/src/config.rs#L167, should the value be `/etc/istio/config/mesh`?

Reply:
the `mesh` key under the configmap is the file name so I do think this is right

Reply:
Yes the `mesh` key is the file name, and I've tested this configmap can be correctly loaded.